daemon/src/engine/null/null_rocket.cpp

```diff
@@ -84,3 +84,4 @@ void Rocket_RegisterProperty( const char*, const char*, bool, bool, const char*
 void Rocket_ShowScoreboard( const char*, bool ) {}
 void Rocket_SetDataSelectIndex( int ) {}
 void Rocket_LoadFont( const char* ) {}
+void Rocket_HideMouse() {}
```

Comment:
Just build all three executables, if the functions are used you'll get link errors.